pilot/pkg/util/sets/string.go

```diff
@@ -57,6 +73,16 @@ func (s Set) Difference(s2 Set) Set {
 	return result
 }
 
+// SupersetOf returns true if s contains all elements of s2
+// For example:
+// s = {a1, a2, a3}
+// s2 = {a1, a2, a3, a4, a5}
+// s.Difference(s2) = {a3}
+// s2.Difference(s) = {a4, a5}
+func (s Set) SupersetOf(s2 Set) bool {
+	return len(s2.Difference(s)) == 0
```

Comment:
Is this right?

// s = {a1, a2, a3}
// s2 = {a1, a2, a4, a5}

Reply:
good catch